pygeo/DVGeometry.py

```diff
@@ -586,6 +589,10 @@ def addGeoDVGlobal(self, dvName, value, func, lower=None, upper=None,
             configurations. The default value of None implies that the design
             variable appies to *ALL* configurations.
         """
+        # if the parent DVGeometry object has a name attribute, prepend it
+        if self.name is not None:
+            dvName = self.name + '_' + dvName
```

Comment:
how strictly do you want this naming convention of compName+dvName? I was doing compName:dvName in dvgeomulti, but I can also change my approach to be consistent. In my implementation, it was dvgeomulti itself tracking the names I believe.